File: php-slim/lib/Model/RequestTimeMapDepartureSearch.php
<?php
/**
 * RequestTimeMapDepartureSearch
 */
namespace OpenAPIServer\Model;

/**
 * RequestTimeMapDepartureSearch
 */
class RequestTimeMapDepartureSearch
{
    
    /** @var string $id */
    private $id;
    
    /** @var \OpenAPIServer\Model\Coords $coords */
    private $coords;
    
    /** @var \OpenAPIServer\Model\RequestTransportation $transportation */
    private $transportation;
    
    /** @var int $travelTime */
    private $travelTime;
    
    /** @var \DateTime $departureTime */
    private $departureTime;
    
    /** @var \OpenAPIServer\Model\RequestTimeMapProperty[] $properties */
    private $properties;
    
    /** @var \OpenAPIServer\Model\RequestRangeNoMaxResults $range */
    private $range;
}

File: fsharp-giraffe-server/OpenAPI/src/model/RequestTimeMapDepartureSearch.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.Coords
open OpenAPI.Model.RequestLevelOfDetail
open OpenAPI.Model.RequestRangeNoMaxResults
open OpenAPI.Model.RequestTimeMapProperty
open OpenAPI.Model.RequestTransportation

module RequestTimeMapDepartureSearch = 

  //#region RequestTimeMapDepartureSearch


  type RequestTimeMapDepartureSearch = {
    Id : string;
    Coords : Coords;
    Transportation : RequestTransportation;
    TravelTime : int;
    DepartureTime : DateTime;
    Properties : RequestTimeMapProperty[];
    Range : RequestRangeNoMaxResults;
    LevelOfDetail : RequestLevelOfDetail;
  }
  //#endregion
  